go.graphics.swing/src/go/graphics/swing/AreaContainer.java

```diff
@@ -26,14 +26,14 @@
 
 import javax.swing.JPanel;
 
-import com.jogamp.opengl.GL2;
-import com.jogamp.opengl.GLAutoDrawable;
-import com.jogamp.opengl.GLCapabilities;
-import com.jogamp.opengl.GLEventListener;
-import com.jogamp.opengl.GLProfile;
-import com.jogamp.opengl.awt.GLCanvas;
-import com.jogamp.opengl.awt.GLJPanel;
-import com.jogamp.opengl.glu.GLU;
+import javax.media.opengl.GL2;
+import javax.media.opengl.GLAutoDrawable;
```

Comment:
So which package should be correct?This was pulled out of the latest jar in maven central (I think..)                                                                                                                                                                                                                                                                                                                                        --David DawsonCEO Simplicity Itself LtdTel +447866011256Skype davidadawsondavid.dawson@simplicityitself.comhttp://www.simplicityitself.com                                                                                                                                                                                                                From: adamjryanSent: Friday, 29 May 2015 18:15To: jsettlers/settlers-remakeReply To: jsettlers/settlers-remakeCc: David DawsonSubject: Re: [settlers-remake] First cut gradle build. (#51)In go.graphics.swing/src/go/graphics/swing/AreaContainer.java:
> @@ -26,14 +26,14 @@
>  
>  import javax.swing.JPanel;
>  
> -import com.jogamp.opengl.GL2;
> -import com.jogamp.opengl.GLAutoDrawable;
> -import com.jogamp.opengl.GLCapabilities;
> -import com.jogamp.opengl.GLEventListener;
> -import com.jogamp.opengl.GLProfile;
> -import com.jogamp.opengl.awt.GLCanvas;
> -import com.jogamp.opengl.awt.GLJPanel;
> -import com.jogamp.opengl.glu.GLU;
> +import javax.media.opengl.GL2;
> +import javax.media.opengl.GLAutoDrawable;

See https://jogamp.org/bugzilla/show_bug.cgi?id=682

—Reply to this email directly or view it on GitHub.